frontend/src/App.vue

```diff
@@ -250,9 +328,36 @@ export default {
       }
       return bpaName;
     },
+    getTitle() {
+      if (this.ux.header.title.prefix) {
+        let pageTitle = (this.title && this.title.trim().length > 0) ? ` : ${this.title}` : "";
+        return `${this.getAgentName} ${pageTitle}`;
+      } else {
+        return this.title;
+      }
+    },
   },
   created() {
-    this.$vuetify.theme.dark = false;
+    if (this.$config.ux) {
+      Object.assign(this.ux, this.$config.ux);
+
+      // Copy the the configuration UX themes, this allows us to change primary color later...
+      if (this.ux.theme.override) {
```

Comment:
Is the override property needed? So you have the local defaults and as soon the web call responds with a section like ux.theme the local section is meant to be written anyway. So long story short shouldn't `if(this.ux.theme)` have the same effect?

Reply:
I've updated to remove the `override` properties and just use the parent property as suggested.